CONTRIBUTING.md

```diff
@@ -167,6 +167,15 @@ features. For more information visit the [Babel documentation site]
 - https://saucelabs.com/beta/dashboard/tests
 - http://developer.samsung.com/remotetestlab/rtlDeviceList.action#
 
+#### Notes
+
+The CSS and JS files that are generated during the build task are only used
+for testing. Because this project doesn't necessarily produce a final product
+it is up to the projects that use it to generate and maintain their own browser
+support config. An example is cfgov-refresh generates specific IE 8 and 9
```

Comment:
`An example…` sentence reads a little awkward. Missing a `, which`?

Reply:
How about:

```
An example of this is [cfgov-refresh](https://github.com/cfpb/cfgov-refresh), which generates specific IE 8 and 9 stylesheets, whereas the Capital Framework build task bundles those with the main stylesheet.
```

Reply:
Since we're talking about the final product, it could read `… [consumerfinance.gov](https://github.com/cfpb/cfgov-refresh) …`